dffml/util/cli/cmds.py

```diff
@@ -111,7 +111,8 @@ class SourcesCMD(CMD):
                 FileSourceConfig(
                     filename=os.path.join(
                         os.path.expanduser("~"), ".cache", "dffml.json"
-                    )
+                    ),
+                    label="default",
```

Comment:
Anywhere where you've explicitly set label, remove it. Unless it's for a testcase to check that data with different labels is successfully living together in the same source. (Please add a testcase for this, so two JSONSource's with the same file and different labels, check that loading and saving them both results in data of both labels being present, repeat for CSVSource when that's ready).

Reply:
Okay :+1: I'll add the test cases once we debug the errors.